hack/make-release-artifacts.sh

```diff
@@ -77,13 +77,19 @@ mk_kubernetes_manifests() {
 
     print_license_header
     print_autogenerated_warning
+
+    echo '# [START gke_release_kubernetes_manifests_microservices_demo]'
```

Comment:
did we test this?

Reply:
so what we expect to happen is, when we cut the next release, these region tags will appear?

Reply:
yup, the next release that happens, whoever runs the `make-release.sh`, the region tags will appear after 
```
# ----------------------------------------------------------
# WARNING: This file is autogenerated. Do not manually edit.
# ----------------------------------------------------------
```
and at the end of the file in both `release/istio-manifests.yaml` and `release/kubernetes-manifests.yaml`

I tested this myself, and changed the released files back before putting in my most recent commit (but anyone can test it by running the make-release-artifacts.sh and seeing the tags there)

Reply:
Thanks for testing this, Christine. :)
I'm glad we removed the tags in the `.yaml` files inside `kubernetes-manifests/`!
I think we can resolve this thread.